config/application.rb

```diff
@@ -40,6 +40,10 @@ class Application < Rails::Application
       require_dependency(c)
     end
 
+    Dir.glob(Rails.root + "app/middlewares/*.rb").sort.each do |file|
+      require file
+    end
```

Comment:
This doesn't actually use the middleware, just loads it so that other files don't need to `require` it explicitly, right?

Reply:
Is this needed with Zeitwerk? I was under the impression everything under /app was autoloaded?

Reply:
Wasn't autoloaded unless I was missing something locally